diagnostic.md

```diff
@@ -14,32 +18,44 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+The each function iterates over a jQuery object, executing a function for each matched element
+$( "ol li" ).each(function( index ) {
+  console.log( index + ": " + $( this ).text() );
+});
```

Comment:
You might have overthought this a bit. There are different ways of doing it heres an example:

`$('ol').children()`

or

`$('ol > li')`